src/commands/schema/push.ts

```diff
@@ -10,9 +10,9 @@ export default class PushSchemaCommand extends SchemaCommand {
       description: "Push the change without a diff or schema version check",
       default: false,
     }),
-    staged: Flags.boolean({
+    unstaged: Flags.boolean({
```

Comment:
i think it might be more elegant to do one of:
- `--skip-staging` / `--skip-staged` (boolean, optional, default false)
- `--target stage` / `--target staged` / `--target active` (string, optional, default stage)

Reply:
I like `--skip-staged,` I'll update.